src/components/notes-listing/note-card/NoteCard.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./NoteCard.scss";
+const NoteCard = () => {
+  return (
+    <section className="mg-b-1">
+      <div class="flex-spBt pd-top-1">
```

Comment:
Replace all occurances of `class` with `className`

Reply:
Yes .